Mobile/starter_project_matador/lib/features/example/presentation/screen/write_article_screen.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+
+class WriteArticlePage extends StatelessWidget {
+  const WriteArticlePage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final Size screenSize = MediaQuery.of(context).size;
+    final bool isPortrait = MediaQuery.of(context).orientation == Orientation.portrait;
+
+    return Scaffold(
+      appBar: AppBar(
+        elevation: 0,
+        backgroundColor: Colors.white,
+        leading: IconButton(
+          icon: const Icon(Icons.arrow_back, color: Colors.black),
+          onPressed: () {},
+        ),
+        title: const Text(
```

Comment:
You didn't even use the widget "ArticleTitle" here so removing the widget would be nice like I said above